proof/refine/ARM/TcbAcc_R.thy

```diff
@@ -5469,5 +5506,15 @@ lemma tcbSchedEnqueue_obj_at':
   apply simp
   done
 
+lemma setThreadState_not_runnable_valid_queues:
```

Comment:
You can delete this and just use `setThreadState_valid_queues` which I updated with the remove runnable' stuff